### PR TITLE
Increase disk size for kubernetes master.

### PR DESCRIPTION
**Pull request by fgrzadkowski**
This improves request latency (#4521).

@wojtek-t

### CODE REVIEW

File: cluster/gce/config-default.sh [hunk 1]
```diff
@@ -21,6 +21,8 @@ ZONE=${KUBE_GCE_ZONE:-us-central1-b}
 MASTER_SIZE=${MASTER_SIZE:-n1-standard-1}
 MINION_SIZE=${MINION_SIZE:-n1-standard-1}
 NUM_MINIONS=${NUM_MINIONS:-4}
+MASTER_DISK_TYPE=pd-standard
+MASTER_DISK_SIZE=${MINION_DISK_SIZE:-100GB}
```
Comment (wojtek-t) [inline cluster/gce/config-default.sh]: s/MINION/MASTER/

File: cluster/gce/config-test.sh [hunk 2]
```diff
@@ -21,6 +21,8 @@ ZONE=${KUBE_GCE_ZONE:-us-central1-b}
 MASTER_SIZE=${MASTER_SIZE:-g1-small}
 MINION_SIZE=${MINION_SIZE:-g1-small}
 NUM_MINIONS=${NUM_MINIONS:-2}
+MASTER_DISK_TYPE=pd-standard
+MASTER_DISK_SIZE=${MINION_DISK_SIZE:-100GB}
```
Comment (wojtek-t) [inline cluster/gce/config-test.sh]: s/MINION/MASTER/

Comment (fgrzadkowski) [inline cluster/gce/config-test.sh]: Done.

File: cluster/gce/config-default.sh (hunk 1 above)
Comment (fgrzadkowski) [inline cluster/gce/config-default.sh]: Done.

**Comment by wojtek-t** [comment]
LGTM

**Comment by bprashanth** [comment]
@fgrzadkowski for my own education, how did this help? IIUC etcd stores everything in memory. I suppose it helped snapshotting, but that should just be transient?

**Pull request merged**